app/controllers/admin/topics_controller.rb

```diff
@@ -27,7 +27,7 @@
 class Admin::TopicsController < Admin::BaseController
 
   before_action :verify_agent
-  before_action :fetch_counts, :only => ['index','show', 'update_topic', 'user_profile']
+  before_action :fetch_counts, :only => ['index','show', 'update_topic', 'user_profile', 'refresh_list', 'refresh_topic']
```

Comment:
Should the new hash syntax not be preferred? `only: [:index, :show, :update_topic]`